CONTRIBUTING.md

```diff
@@ -229,6 +229,20 @@ Run script to download dashboards to `./dashboards` folder
 node scripts/download_dashboards.mjs
 ```
 
+## Contributing to Documentation
+
+When submitting PRs for documentation updates, build and run the documentation locally to ensure functionality before submission. To build the documentation locally, ensure you have Python installed, then simply execute `yarn build:docs` 
```

Comment:
From the new commands introduced in #6296 , one needs to call `yarn docs:build && yarn docs:serve` to build and serve local docs. 

`yarn docs:install` to install python dependencies for first-timers.